src/main/kotlin/org/bibletranslationtools/fetcher/application/SourceAudioRepository.kt

```diff
@@ -0,0 +1,27 @@
+package org.bibletranslationtools.fetcher.application
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import kotlin.streams.toList
+import org.bibletranslationtools.fetcher.data.Language
+import org.bibletranslationtools.fetcher.domain.LanguageCatalog
+
+class SourceAudioRepository(
```

Comment:
I think this is a use case and its name should not be named "Repository".

https://medium.com/slalom-build/clean-architecture-with-java-11-f78bba431041
This is a good example I found that represented a use case with the repository passed into it.

Reply:
How about SourceAudioContent? We then can use: SourceAudioContent().getLanguages()